### ISSUE TITLE
Duplicates of pages disappeared after reloading

### ISSUE THREAD

**Issue by zhartus** [uxbox/uxbox#1377]
**Describe the bug**
Duplicate pages (edited or not) are not saved.

**To Reproduce**
1.  Create page with some artboards.
2.  Duplicate this page.
3.  Work hard on the new version of the page all day long (this step is optional, but it allows you to better feel the drama of the problem).
4.  Refresh the page and lose all your work. The edited duplicate page just disappears. I am shocked and take anti-anxiety drugs (a joke, but close to the truth).

**Expected behavior**
Obviously, I hope my work doesn't disappear as it does.

**Desktop:**

-   OS: Ubuntu 20.04
-   Browser: Firefox
-   Version 94.0

**Environment:**
https://design.penpot.app

**Additional context**

I tried to repeat the same thing in other documents and now I started to notice the red inscription "Error on saving".
It is noteworthy that it appears only after the duplicate page has been created. In any other operation, the changes are saved normally.
And by the way, I did not notice this inscription before when working on a real project (work on which I lost), although I have a habit of looking into this corner and checking if everything is fine. And as far as I remember, everything was supposedly good.

**Comment by zhartus** [comment]
I found a temporary workaround for this problem. Instead of duplicating the page, you can create a new blank page and copy all the artboards you want to it.
But the page duplication functionality still looks completely broken and even dangerous, since the problem can be noticed too late (as it happened in my case).

**Comment by niwinz** [comment]
Can you send us the link to the workspace?, that will help us understand the underlying problem, The duplicate page is not broken, but looks broken on this specific file. And the link will help us to find the file in the database and check what happens. Thanks

**Comment by zhartus** [comment]
@niwinz 
Hi Andrey,
Thank you for your attention to the problem.
I sent you a link to the workspace by email.

**Comment by niwinz** [comment]
I found the issue, for the curiosity, the problem is the **Auto** value on saved grids. I'm currently working on fixing it.

**Comment by zhartus** [comment]
@niwinz 
Great, thank you for your work!

**Comment by niwinz** [comment]
FYI, we got a fix, but we need to test it for not break other things. If everything goes ok, probably tomorrow it will be deployed. But I can't promise it.

**Comment by niwinz** [comment]
The fix is deployed, Can you check if it works from your side (please refresh the page for load the latest version)

**Comment by zhartus** [comment]
@niwinz 
Yes, it works now. Thank you very much!

**Comment by niwinz** [comment]
awesome
